src/tests/multihost/ipa/conftest.py

```diff
@@ -165,6 +192,33 @@ def restoresssdconf():
 # ====================  Class Scoped Fixtures ================
 
 
+@pytest.fixture(scope='class')
+def install_list_subid_ranges(session_multihost, request):
+    """
+    Install necessary packages
+    """
+    session_multihost.client[0].run_command("yum --enablerepo=rhel-CRB install"
+                                            " -y shadow-utils*")
+    session_multihost.client[0].run_command("yum install -y shadow-utils*")
+    session_multihost.client[0].run_command("yum install -y gcc")
+    with pytest.raises(subprocess.CalledProcessError):
+        session_multihost.client[0].run_command(f"grep subid "
+                                                f"/etc/nsswitch.conf")
+    session_multihost.client[0].transport.put_file(os.getcwd() +
+                                                   '/data/list_subid_ranges.c',
+                                                   '/tmp/list_subid_ranges.c')
+    session_multihost.client[0].run_command("gcc /tmp/list_subid_ranges.c "
+                                            "-lsubid -o  "
+                                            "/tmp/list_subid_ranges")
+
+    def remove():
+        """ Remove file """
+        session_multihost.client[0].run_command("rm -vf "
+                                                "/tmp/list_subid_ranges")
```

Comment:
For consistency it also makes sense to delete '/tmp/list_subid_ranges.c'

Reply:
Fixed